samples/samples/async_client_sample.py

```diff
@@ -0,0 +1,648 @@
+
+     print("\nStarting task: ", task_num)
+
+     """Request object for delete session."""
+     request = DeleteSessionRequest(
+         name=session_name,
+     )
+
+     """Delete the session."""
+     await async_client.delete_session(request = request)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client delete_session]
+
+# [START async client list_sessions]
+async def list_sessions(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     """Request object for list session."""
+     request = ListSessionsRequest(
+         database = database_object.name,
+     )
+
+     """Get the list of sessions."""
+     response = await async_client.list_sessions(request = request)
+
+     """Prints the session."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client get_session]
+
+
+# [START async client get_session]
+async def get_session(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     # Create the session object associated with the database object
+     session_object = session.Session(database_object)
+
+     """Create the session."""
+     session_object.create()
+
+     """Request object for get session."""
+     request = GetSessionRequest(
+         name = session_object.name,
+     )
+
+     """Get the session."""
+     response = await async_client.get_session(request = request)
+
+     """Prints the session got."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client get_session]
+
+
+# [START async client batch_create_session]
+async def batch_create_session(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     """Request object for batch create session."""
+     request = BatchCreateSessionsRequest(
+         database = database_object.name,
+         session_count=1420,
+     )
+
+     """Creates batch sessions."""
+     response = await async_client.batch_create_sessions(request = request)
+
+     """Prints the sessions batchwise."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client batch_create_session]
+
+
+# [START async client create_session]
+async def create_session(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     """Request object for create session."""
+     request = CreateSessionRequest(
+         database = database_object.name,
+     )
+
+     """Creates a session."""
+     response = await async_client.create_session(request = request)
+
+     """Prints the session."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client create_session]
+
+
+async def main():
+    parser = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+    parser.add_argument("--instance_id", help="Your Cloud Spanner instance ID.")
```

Comment:
Can we provide a default value for instance as well? (as we have provided it for database_id)

Or we could skip the default value for database_id as well.

Reply:
Done